api/src/models/service_config/mod.rs

```diff
@@ -339,6 +366,27 @@ mod tests {
     use secstr::SecUtf8;
     use serde_json::from_value;
 
+    fn desecure_files(
```

Comment:
Please, remove. You can compare the results in the test by creating `SecStr` instead of `String` in the assertions.